src/EnergyPlus/SolarCollectors.cc

```diff
@@ -2164,6 +2137,37 @@ namespace SolarCollectors {
             VentCavIndex = CavNum;
         }
     }
+    void CollectorData::oneTimeInit(EnergyPlusData &state)
+    {
+
+        if (this->MyOneTimeFlag) {
+            this->setupOutputVars(state);
+            this->MyOneTimeFlag = false;
+        }
+
+        if (this->SetLoopIndexFlag) {
+            if (allocated(state.dataPlnt->PlantLoop)) {
```

Comment:
Another function with two different booleans that will soon be deleted and an eliminated check on the plant allocation.  Nice.